### ISSUE TITLE
After upgrading the vscode1.29 version, can't read data from multiple threads in pytorch.

### ISSUE THREAD

**Issue by ErenTuring** [Microsoft/vscode#63415]
After upgrading the vscode 1.29 version, the code that I was able to run normally suddenly reported an error.
I am in the pytorch 0.41 environment. Specifically, the place I found the error was the data reading part.
`
    from torch.utils.data import DataLoader
`
Once the num_workers in the DataLoader are set to be greater than 0, that is, multi-threaded read.
The program will be interrupted and prompted.

-----------------------------------------------------------------------------------------------------
After upgrading the vscode 1.29.1 version, under the windows system, the above problems no longer exist. But under the Ubuntu16 system, if multiple threads read data, it will get stuck. While I test the use of the terminal directly run my code, it can run normally.
An example code：
`
from torch.utils.data import DataLoader
from utils.dataset import MyDataset

root = 'XXXX'  # data_path

def train():
    train_data = MyDataset(root)  # Self defining Data class
    train_dataloader = DataLoader(
            train_data, 2, shuffle=True, num_workers=opt.num_workers)

    for ii, (image, label) in enumerate(train_dataloader):
        print(image.size())

if __name__ == '__main__':
    train()
`

**Comment by weinand** [comment]
VS Code 1.9 is from January 2017. Are you sure that you are talking about version "1.9"?

**Comment by weinand** [comment]
Please file this issue against the Python extension.
